### PR TITLE
fix guest group

### PR DESCRIPTION
**Pull request by BrokenBulb78**

### CODE REVIEW

File: multiplayer_server/Player.php [hunk 1]
```diff
@@ -1312,8 +1312,8 @@ public function becomeServerOwner()
     public function becomeGuest()
     {
         $this->guest = true;
-        $this->group = 1;
-        $this->write('setGroup`1');
+        $this->group = 0;
+        $this->write('setGroup`0');
```
Comment (bls1999) [inline multiplayer_server/Player.php]: You can take out lines 1315 and 1316 altogether.

**Comment by famelegend1** [comment]
"No description provided."

File: multiplayer_server/Player.php (hunk 1 above)
Comment (BrokenBulb78) [reply multiplayer_server/Player.php]: Oh, I thought it was needed

Comment (bls1999) [reply multiplayer_server/Player.php]: It won't hurt anything, but it's already taken care of in the `process_register_login()` function in `process_fns.php`.

**Pull request merged**